File: 2026/01/05/main.java
import java.io.*;

public class Main {
    public static void main(String[] args) throws IOException{
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        char[] str = br.readLine().toCharArray();
        int count = 0;
        for(char c : str){
            int ret = 0;
            if(c < 'a'){
                ret += 26;
                ret += c - 'A' + 1;
            } else {
                ret += (c - 'a' + 1);
            }
            count += ret;
        }

        if(isPrime(count)){
            System.out.print("It is a prime word.");
        } else {
            System.out.print("It is not a prime word.");
        }



    }

    static boolean isPrime(int num){
        if(num == 1){
            return true;
        }

        for(int i=2; i<= Math.sqrt(num); i++){
            if(num % i == 0) return false;
        }

        return true;
    }
}

File: 2026/01/05/main.py
def is_prime(n):
    if n == 1:
        return True

    for i in range(2, int(n ** 0.5)+1):
        if n % i == 0:
            return False

    return True

s = input()
count = 0
for c in s:
    ret = 0
    if ord(c) < ord('a'):
        ret += 26
        c = c.lower()
    ret += (ord(c) - ord('a')) + 1
    count += ret

print("It is a prime word." if is_prime(count) else "It is not a prime word.")

